examples/community/composable_stable_diffusion.py

```diff
@@ -341,7 +341,7 @@ def decode_latents(self, latents):
         image = self.vae.decode(latents).sample
         image = (image / 2 + 0.5).clamp(0, 1)
         # we always cast to float32 as this does not cause significant overhead and is compatible with bfloat16
-        image = image.cpu().permute(0, 2, 3, 1).float().numpy()
+        image = image.cpu().permute(0, 2, 3, 1).float().detach().numpy()
```

Comment:
I'm not sure why we're adding this? Latent decoding happens in call which is always wrapped in no grad :)